src/architecture/_GeneralModuleFiles/swig_conly_data.i

```diff
@@ -80,11 +80,9 @@
 
 %enddef
 ARRAYASLIST(double)
-ARRAYASLIST(int)
 ARRAYASLIST(float)
-ARRAYASLIST(unsigned int)
 
-%define ARRAYINTASLIST(type)
+%define ARRAYINTASLIST(type, fromfunc, asfunc)
```

Comment:
Now that we have this function with the from and as func, do you think we could also use it for the double types? Seems like the code is pretty much the same for `ARRAYINTASLIST` and `ARRAYASLIST`.

Reply:
Yeah that would make sense. Right now the floating point version has an explicit cast to double, but it shouldn't be necessary since C will implicit cast that, so I will try to merge them